internal/command/migrate_to_v2/troubleshoot.go

```diff
@@ -183,20 +181,7 @@ func (t *troubleshooter) hasMigrated(ctx context.Context) bool {
 		}
 	}
 
-	// Look for a release created by admin-bot@fly.io
-	releases, err := client.GetAppReleasesMachines(ctx, t.app.Name, "", 25)
-	if err != nil {
-		return false
-	}
-	for _, release := range releases {
-		// Technically, I don't think this is the only time we could use admin-bot@fly.io,
-		// but we use it infrequently and soon we'll be done dealing with this,
-		// so it's probably an acceptable way to determine this for now.
-		if release.User.Email == "admin-bot@fly.io" {
-			return true
-		}
-	}
-	return false
+	return true
```

Comment:
This makes me think the entire function should just be
```
return t.app.PlatformVersion != appconfig.NomadPlatform
```

Reply:
Yeah, that's what I was about to say. It used to be more important that we got this perfect, but it's a lot more graceful now so we can just say "if not nomad, keep going"